src/main/java/frontend/ContainerPanel.java

```diff
@@ -42,7 +42,7 @@ private void addComponents()
     {
         //Label myLabel = new Label("Hello World!");
```

Comment:
several lines of commented code. is this needed? is it dead? lets not have cruft this early on.